operations/rack_resiliency/manage_critical_services.md

```diff
@@ -0,0 +1,394 @@
+# Editing/Deleting criticalservices from static configmap (rrs-mon-static)
+
+## Overview
+The rrs-mon-static ConfigMap in the rack-resiliency namespace contains configurations for critical services.
+
+Procedure to delete a criticalservice from the configmap
+Verify the ConfigMap exists:: kubectl get configmaps -n rack-resiliency
+
+### Example
+
+```bash
+kubectl get configmaps -n rack-resiliency 
+NAME                 DATA   AGE
+istio-ca-root-cert   1      6d21h
+kube-root-ca.crt     1      6d21h
+rrs-mon-dynamic      2      3d4h
+rrs-mon-static       11     3d4h
+```
+
+Our list of criticalservices is present in rrs-mon-static.
+To verify if the list is present run: kubectl describe configmap rrs-mon-static -n rack-resiliency
+
+### Example
+
+```bash
+kubectl describe configmap rrs-mon-static -n rack-resiliency 
+Name:         rrs-mon-static
+Namespace:    rack-resiliency
+Labels:       <none>
+Annotations:  <none>
+```
+
+### Data
+
+#### ceph_pre_monitoring_delay:
+60
+
+#### critical-service-config.json
+
+```json
+{
+  "critical_services": {
+    "cilium-operator": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "coredns": {
+      "namespace": "kube-system",
+      "type": "Deployment"
+    },
+    "cray-power-control-bitnami-etcd": {
+      "namespace": "services",
+      "type": "StatefulSet"
+    },
+    "cray-powerdns-manager": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-sls": {
+      "namespace": "services",
+      "type": "Deployment"
+    },
+    "cray-spire-postgres": {
+      "namespace": "spire",
+      "type": "StatefulSet"
+    },
+    ...
+}
+```
+
+#### k8s_monitoring_total_time:
+600
+
+#### k8s_pre_monitoring_delay:
+40
+
+#### ceph_monitoring_polling_interval:
+60
+
+#### default_message_level:
+debug
+
+#### k8s_monitoring_polling_interval:
+60
+
+#### last_updated_timestamp:
+2025-06-30T04:49:35.298283Z
+
+##### log_dir:
+/var/log/rr
+
+#### unit_of_time:
+seconds
+
+#### ceph_monitoring_total_time:
+600
+
+#### BinaryData
+Events:  <none>
+
+In order to delete the service from the configmap use the following command: 
+
+```bash
+kubectl edit configmap rrs-mon-static -n rack-resiliency
+```
+
+You will see the following snippet:
+
+```json
```

Comment:
This is actually yaml, not json.  Only the embedded file is json.  So either `yaml` or just `text`.